dll/cpl/openglcfg/lang/es-ES.rc

```diff
@@ -0,0 +1,28 @@
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```

Comment:
...and this one is not en-US 😆

Reply:
Ouch! I commit a mistake with the files, sorry.

Reply:
Fixed it for you :)